python/openassetio/hostAPI/Manager.py

```diff
@@ -966,9 +966,8 @@ def preflight(self, targetEntityRefs, entitySpecs, context):
         @param targetEntityRefs `List[str]` The entity references to
         preflight prior to registration.
 
-        @param entitySpecs `List[`
-            specifications.EntitySpecification `]`
-        A description of each entity that is being published.
+        @param traitSet `Set(str)` The @needsref traitSet of the
```

Comment:
[consider][minor] Here and in `ManagerInterface`, for Python 3 type hint style
```suggestion
        @param traitSet `Set[str]` The @needsref traitSet of the
```